components/director/pkg/inputvalidation/directive_test.go

```diff
@@ -0,0 +1,62 @@
+package inputvalidation
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type testStruct struct {
+	returnErr error
+}
+
+func (ts testStruct) Validate() error {
+	return ts.returnErr
+}
+
+func TestDirective_Validate(t *testing.T) {
+	validatorDirective := NewDirective()
+	testErr := errors.New("testError")
+	ctx := context.TODO()
+	t.Run("Success", func(t *testing.T) {
```

Comment:
to be consistent with other tests

```suggestion
	t.Run("success", func(t *testing.T) {
```